tests/lsp_features/code_actions.zig

```diff
@@ -363,6 +363,166 @@ test "ignore autofix comment whitespace" {
     );
 }
 
+const importDiagnostic = types.Diagnostic{
+    .range = .{
+        .start = .{ .line = 1, .character = 0 },
+        .end = .{ .line = 2, .character = 0 },
+    },
+    .severity = .Hint,
+    .code = .{ .string = "unorganized_import" },
+    .source = "zls",
+    .message = "unorganized @import statement",
+};
+
+test "organize imports" {
```

Comment:
The calls to `testDiagnostic` can be moved into separate test decls. Here are some suggestions:
```zig
test "organize imports" { ... }
test "organize imports - preserve doc comments" { ... }
test "organize imports - top-level doc-comment" { ... }
test "organize imports - duplicate builtin import" { ... }
```